cohort_schedules/C07_classroom.md

```diff
@@ -1,39 +1,126 @@
-| Mar 13  | Week 06 | Sinatra & SQL
-| Mar 20  | Week 07 | _BREAK_
-| Mar 27  | Week 08 | Intro to Rails
-| Apr 3   | Week 09 | Rails II, Unit testing
-| Apr 10  | Week 10 | Rails III, OAuth, Agile, PM Skills
-| Apr 17  | Week 11 | Controller filters, advanced validations, AR callbacks
-| Apr 24  | Week 12 | Individual Project - No new topics
-| May 1   | Week 13 | Rails Using APIs
-| May 8   | Week 14 | Rails Creating & Using APIs
-| May 15  | Week 15 | Intro to JavaScript
-| May 22  | Week 16 | jQuery and AJAX
-| May 29  | Week 17 | Intro to Backbone
-| Jun 5   | Week 18 | Interviews
-| Jun 12  | Week 19 | Intermediate Backbone
-| Jun 19  | Week 20 | Advanced Backbone
-| Jun 26  | Week 21 | _BREAK_
-| Jul 3   | Week 22 | Capstone Project
-| Jul 10  | Week 23 | Capstone Project
-| Jul 17  | Week 24 | Capstone Project
-| Jul 24  | Week 25 | Capstone Project Presentations
-| Jul 31  | Week 26 | _BREAK_
-| Aug 7   | Week 27 | __INTERNSHIP BEGINS__
+# Ada Developers Academy Cohort 7 Syllabus
+
+## Overview
+
+Ada Developers Academy teaches full-stack web development, allowing students to choose what specialty of software development they want to pursue in their post-Ada careers. We cover HTML/CSS, JavaScript, Ruby on Rails, and computer science fundamentals. Our [complete curriculum](https://github.com/Ada-Developers-Academy/textbook-curriculum/) is online and open-source.
+
+We focus on teaching skills that are readily transferrable from one technology stack to another, making Ada students adaptable and flexible candidates for all types of developer positions. Students have been successfully placed in internships and jobs working in Ruby, Java, JavaScript, C++, C#, Objective-C and Python, among other languages and frameworks.
+
+Major topics of study include:
+
+| Writing Software        | Being a Developer
+|-------------------------|-------------------
+| Ruby                    | Leadership & Inclusion
+| HTML & CSS              | Test-Driven Development
+| Ruby on Rails           | Pair Programming
+| Web APIs                | Agile Practices
+| JavaScript              | Networking
+| CS Fundamentals         | Career Readiness
+
+## Expected Outcomes
+
+### Ruby
+
+Ruby is a dynamically typed, interpreted, object-oriented language. In addition to serving as a great introduction to programming, Ruby is the foundation of the Ruby on Rails framework, and is used for most of our CS Fundamentals curriculum.
+
+Students should be able to:
+
+* Write complex command line programs in Ruby
+* Use object-oriented programming principals to build scalable, maintainable programs
+* Process and build relations between data from files
+* Verify code from the outset using test-driven development
+
+### HTML & CSS
+
+HTML and CSS are two of the core technologies on which the web is built, allowing web developers to structure content in a hierarchy and specify how it should be presented to the user. Both are foundational for much of the rest of the course.
```

Comment:
Maybe something more like "Both are foundational for web developers" instead of focusing on the coursework